nmostesting/ControllerTest.py

```diff
@@ -167,6 +165,10 @@ def execute_tests(self, test_names):
         self.post_tests_message()
 
     async def get_answer_response(self, timeout):
+        # Add API processing time to specified timeout
+        # Otherwise, if timeout is None then disable timeout mechanism
+        timeout = timeout + (2 * CONFIG.API_PROCESSING_TIMEOUT) if timeout else None
```

Comment:
This treats 0 the same as None, but none of the other code does. For consistency some `or None` should be added in other places _or_ this could be changed to the ugly:

```suggestion
        timeout = timeout + (2 * CONFIG.API_PROCESSING_TIMEOUT) if timeout is not None else None
```